src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+    &:hover {
```

```suggestion
    transition-duration: 300ms;
    
    &:hover {
```
use transition shorthand syntax